Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -382,80 +165,92 @@ void Server::start()
-    Base::Logger::FileLogger::getInstance().log("[SERVER] Stopped!", Base::Logger::LogLevel::INFO);
+    _threads.clear();
+
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "[SERVER] Stopped!",
+        Base::Logger::LogLevel::INFO
+    );
 }
 
 void Server::waitForClientConnection()
+{   
+    _acceptor.async_accept([this](const std::error_code error, asio::ip::tcp::socket socket) { acceptingClientConnection(error,socket); });
+}
+
+void Server::acceptingClientConnection(const std::error_code& error, asio::ip::tcp::socket& socket)
 {
-    mAcceptor.async_accept([this](std::error_code error, asio::ip::tcp::socket socket) {
-        if (!error)
+    if (!error)
+    {
+        std::ostringstream outputPoint;
+        outputPoint << socket.remote_endpoint();
+
+        Base::Logger::FileLogger::getInstance().log
+        (
+            "[SERVER] New Connection: " + outputPoint.str(),
+            Base::Logger::LogLevel::INFO
+        );
+
+        auto newConnection = 
+            std::make_shared<Connection>(Connection::OwnerType::SERVER, _context, std::move(socket), _incomingMessagesQueue);
+
+        if (onClientConnect(newConnection))
         {
-            std::ostringstream out;
-            out << socket.remote_endpoint();
+            _connectionsPointers.emplace_back(std::move(newConnection));
+            _connectionsPointers.back()->connectToClient(_IDCounter++);
+
             Base::Logger::FileLogger::getInstance().log
             (
-                "[SERVER] New Connection: " + out.str(), 
+                "[" + std::to_string(_connectionsPointers.back()->getID()) 
+                + "] Connection Approved", 
                 Base::Logger::LogLevel::INFO
             );
-
-            std::shared_ptr<Connection> newConnection =
-                std::make_shared<Connection>(Connection::OwnerType::SERVER, mContext, std::move(socket), mIncomingMessagesQueue);
-
-            /// @todo This function always return true
-            if (onClientConnect(newConnection))
-            {
-                mConnectionsPointers.push_back(std::move(newConnection));
-
-                mConnectionsPointers.back()->connectToClient(mIDCounter++);
-
-                Base::Logger::FileLogger::getInstance().log
-                (
-                   "[" + std::to_string(mConnectionsPointers.back()->getID()) + "] Connection Approved",
-                    Base::Logger::LogLevel::INFO
-                );
-            }
-            else
-            {
-                Base::Logger::FileLogger::getInstance().log
-                (
-                    "[-----] Connection Denied", 
-                    Base::Logger::LogLevel::INFO
-                );
-            }
         }
         else
         {
             Base::Logger::FileLogger::getInstance().log
             (
-                "[SERVER] New Connection Error: " + error.message(),
-                Base::Logger::LogLevel::ERR
+                "[-----] Connection Denied", 
```

Comment:
Why dashes?

Reply:
![image](https://user-images.githubusercontent.com/70116007/169314897-9680be3b-36ed-47bd-a639-38adaac57a81.png)

I don't know, i don't toucheed it. 
It was in dev branch.

Reply:
> 

I'm resolve it in my branch some of result
![image](https://user-images.githubusercontent.com/70116007/169315744-2e557a70-fb6f-404e-9ba3-d6abf06a33ea.png)